src/popover/__tests__/popover.test.tsx

```diff
@@ -98,7 +98,7 @@ describe('Dismiss button', () => {
       const wrapper = renderPopover({ children: 'Trigger', content: 'Popover', renderWithPortal });
       wrapper.findTrigger().click();
       act(() => {
-        document.dispatchEvent(new MouseEvent('mousedown'));
+        document.dispatchEvent(new MouseEvent('mousedown', { bubbles: true }));
```

Comment:
Why was `{ bubbles: true }` not necessary before and now it is?

Reply:
OK. This should have no effect on real users because native events bubble up by default, right? (So actually this is a better simulation of a real world interaction)

I would just add a comment in the tests to explain that.

Reply:
> Well, I only changed the tests :P

Yeah well, my question was not very well formulated —what I wondered is whether this change being necessary in the test was a sign that something changed that could affect the user's experience too. But from what you say the test was also not firing the event correctly before this change, so in fact the bubbling would have been necessary already as well.

> Pushing back slightly cause I don't know what I'd put in a comment that explains more than what's obvious

Fair enough, no big concerns.